partner_compassion/views/res_partner_view.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<record id="act_partner_survey_input" model="ir.actions.act_window">
```

Comment:
This should be moved in survey_user_input_action.xml file (and included in manifest)